### ISSUE TITLE
Track PDF access to Dataset pages as a Dataset View in Google Analytics

### ISSUE THREAD

**Issue by ashepherd** [BCODMO/pdfgenerator#2]

**Comment by ashepherd** [comment]
Yes, we should track these Events, and we should investigate if we can have this service report the event to Google Analytics

@shravani-whoi, let's discuss

**Comment by shravani-whoi** [comment]
Sounds good. Will look in to Google Analytics as a next step

**Comment by ashepherd** [comment]
I've heard that you can trigger events in GA from code, but I've never done that. I can show you what events we are tracking now and how this would slot in.

**Comment by shravani-whoi** [comment]
Sure let's do this as soon as I have the draft design/spike ready, so that we can fit it in & test it out
